test/gtest/uct/test_md.cc

```diff
@@ -700,7 +700,8 @@ UCS_TEST_SKIP_COND_P(test_md, invalidate, !check_caps(UCT_MD_FLAG_INVALIDATE))
     free(ptr);
 }
 
-UCS_TEST_SKIP_COND_P(test_md, dereg_bad_arg, !check_caps(UCT_MD_FLAG_REG))
+UCS_TEST_SKIP_COND_P(test_md, dereg_bad_arg,
+                     !check_reg_mem_type(UCS_MEMORY_TYPE_HOST))
```

Comment:
why is it needed?

Reply:
it failed to register memory on CUDA and test failed
will improve test later to cover all supported memory types in MD

Reply:
what is the error? could it be some real issue with invalidation impl ?

Reply:
it is failed on line 713 - status is not OK

Reply:
i see, 10x